yarn-project/p2p/src/services/peer-scoring/peer_scoring.ts

```diff
@@ -4,11 +4,21 @@ import { median } from '@aztec/foundation/collection';
 import { type P2PConfig } from '../../config.js';
 
 const DefaultPeerPenalties = {
-  [PeerErrorSeverity.LowToleranceError]: 2,
+  [PeerErrorSeverity.LowToleranceError]: 50,
```

Comment:
think you need to merge with `master`, this change went in already, right?

Reply:
ah yes correct! I must have applied the commit on both branches, merged now